Formula/srcml.rb

```diff
@@ -0,0 +1,223 @@
+class Srcml < Formula
+  desc "convert source code to XML"
+  homepage "http://www.srcml.org/"
+  url "http://131.123.42.38/lmcrs/beta/srcML-src.tar.gz"
```

Comment:
The software is quite stable. It is produced by the NSF grant for quite a few years now. Earlier, I was asked to provide the URL from the official website http://srcml.org, hence it is with the URL. The version number is 0.9.5, I can add it back to the formula.